inbm/dispatcher-agent/dispatcher/fota/fota.py

```diff
@@ -80,6 +81,8 @@ def __init__(self,
 
         self.__signature = parsed_manifest['signature']
         self._hash_algorithm = parsed_manifest['hash_algorithm']
+        if self._hash_algorithm is None:
+            self._hash_algorithm = DEFAULT_HASH_ALGORITHM
```

Comment:
**Suggestion:** Instead of checking if self._hash_algorithm is None and then setting it to DEFAULT_HASH_ALGORITHM, use the get method with a default value when retrieving hash_algorithm from parsed_manifest. This simplifies the code and avoids the need for an additional conditional check. [enhancement]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        self.__signature = parsed_manifest['signature']
        self._hash_algorithm = parsed_manifest.get('hash_algorithm', DEFAULT_HASH_ALGORITHM)
```